### PR TITLE
Fix getting transforms of inputSource after XR session is ended

### PR DESCRIPTION
**Pull request by Maksims**
Fix error with getting transforms on input sources when XR session is ended.

I confirm I have signed the [Contributor License Agreement](https://docs.google.com/a/playcanvas.com/forms/d/1Ih69zQfJG-QDLIEpHr6CsaAs6fPORNOVnMv5nuo0cjk/viewform).

### CODE REVIEW

File: src/xr/xr-input-source.js [hunk 1]
```diff
@@ -224,6 +224,9 @@ Object.assign(pc, function () {
     XrInputSource.prototype._updateTransforms = function () {
         var dirty;
```
Comment (willeastcott) [inline src/xr/xr-input-source.js:225]: Any reason to keep `dirty`?

File: src/xr/xr-input-source.js [hunk 2]
```diff
@@ -224,6 +224,9 @@ Object.assign(pc, function () {
...
 
+        if (! this._manager.active)
+            return;
+
         if (this._dirtyLocal) {
             dirty = true;
```
Comment (willeastcott) [inline src/xr/xr-input-source.js:231]: And this occurrence?

File: src/xr/xr-input-source.js (hunk 1 above)
Comment (Maksims) [reply src/xr/xr-input-source.js:225]: Yes, it uses it slightly bellow to decide to do transform update or not.

File: src/xr/xr-input-source.js (hunk 2 above)
Comment (Maksims) [reply src/xr/xr-input-source.js:231]: It happens when session is ended, but due to WebXR API async nature of session end event, it is possible to call `app.xr.end()` but then getting actual callback of XRSession end later. Although camera pointer is removed immediately on `end` call. So input sources are not removed immediately as they are removed by XRSession events.

We can start removing them by hand when `end` is manually executed.

**Comment by willeastcott** [comment]
Hey @Maksims - is this ready to be merged?

**Pull request merged**